build.sbt

```diff
@@ -2,7 +2,7 @@ import de.johoop.jacoco4sbt.XMLReport
 
 val driverVersion = "3.0.0-SNAPSHOT"
 val asyncHttpClientVersion = "1.9.39"
-val guavaVersion = "19.0"
+val guavaVersion = "21.0"
```

Comment:
This seems like an unfortunate necessity to get Guava up to Java 8 support, just before I remove it. :)